### PR TITLE
[#540] Build Summoner with 9.0

### LINKED ISSUE #540: Build Summoner with 9.0

### PR DESCRIPTION
**Pull request by vrom911**
Resolves #540

### CODE REVIEW

**Comment by vrom911** [comment]
Currently waiting for https://github.com/jtdaugherty/brick/issues/316 and https://github.com/haskell-hvr/microaeson/pull/7

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,70 +1,144 @@
...
     - name: Build
       run: |
-        cabal v2-update
-        cabal v2-build all --enable-tests --enable-benchmarks
+        cabal build all
 
     - name: Unit and property tests
       run: |
-        cabal v2-test all --enable-tests --test-show-details=direct
+        cabal test all --enable-tests --test-show-details=direct
```
Comment (chshersh) [inline .github/workflows/ci.yml:80]: I think this is not needed as it's already in `configure`
```suggestion
        cabal test all
```

**Review by chshersh — APPROVED** [review]
Looks awesome! 💯

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,70 +1,144 @@
...
 
     - if: matrix.ghc == '8.10.3'
```
Comment (chshersh) [inline .github/workflows/ci.yml:82]:
```suggestion
    - if: matrix.ghc == '8.10.4'
```

**Pull request merged**